.github/workflows/deploy.yml

```diff
@@ -7,6 +7,9 @@ name: CI
 on:
   push:
     branches: [ master ]
+  schedule:
+  # Run everyday at 9:00 AM (See https://pubs.opengroup.org/onlinepubs/9699919799/utilities/crontab.html#tag_20_25_07)
+  - cron: "0 9 * * *"
```

Comment:
What time zone is this in?

Reply:
UTC of course.

https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#schedule

Reply:
> UTC of course.
> 
> https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#schedule

Got it.